### ISSUE TITLE
Increase visibility of how to edit configuration.yaml

### ISSUE THREAD

**Issue by Erisa** [home-assistant/home-assistant.github.io#31922]
### Feedback

When adding the Emby integration I was told that I needed to edit `configuration.yaml` and was taken to this page: https://www.home-assistant.io/integrations/emby

On this page it tells me what to edit on `configuration.yaml`, but as a newcomer to Home Assistant it provides no direction on what this file is or where to find it.

Eventually and after asking, I discovered both https://www.home-assistant.io/docs/configuration/ and https://www.home-assistant.io/common-tasks/os/ which shed a lot of light on this subject.

My proposal is that the `{% term` system is used like it is for words in the [glossary](https://github.com/home-assistant/home-assistant.io/blob/current/source/_data/glossary.yml#L232)
It may also help to add a non-technical/high-level explainer to the https://www.home-assistant.io/docs/configuration/ page.

Something like this maybe in glossary:

```yaml
- term: configuration.yaml
  definition: >-
    Some parts of Home Assistant configuration require you to edit `configuration.y`ml. This file contains integrations to be loaded with their configurations.
  link: /docs/configuration/
```

The only reason this is not a PR is because I tried and encountered two critical issues:

1. There would have to be a huge find/replace across the entire docs, when I tried doing this ist caught a lot of unneeded things like comments and it would have to be manually and tediously reviewed for accuracy.
2. My change to the glossary didn't take, I rashly assume it choked on the dot in the term name?

```
  Liquid Exception: Syntax error in tag 'term' while parsing the following options: configuration.yaml Valid syntax: {% term <term> [<text>] %} in /mnt/c/Users/Erisa/Work/home-assistant.io/source/_integrations/acer_projector.markdown
                    ------------------------------------------------
      Jekyll 4.3.3   Please append `--trace` to the `build` command
                     for any additional information or backtrace.
                    ------------------------------------------------
```

### URL

https://www.home-assistant.io/integrations/emby

**Comment by c0ffeeca7** [comment]
Excellent suggestions, @Erisa :+1:

**Comment by c0ffeeca7** [comment]
Closing this as addressed by #32472, #32619, #32635
